openedx/core/djangoapps/user_authn/views/password_reset.py

```diff
@@ -68,6 +68,12 @@ def get_password_reset_form():
         HttpResponse
 
     """
+
+    if not is_testing_environment():
+        from openedx.adg.lms.user_authn_override.password_reset import get_password_reset_form_override
```

Comment:
is it best way to put imports here instead of start of the file. ?

Reply:
This import is the overridden method and also it is used only at one place. I thought it would be better to make a core change at one place as well. What do you say?